app/build.gradle

```
@@ -32,6 +40,9 @@ android {
     }
     buildFeatures {
         viewBinding = true
+
+        //사용할 객체 가져오기 (base_url 감추는 작업)
+        buildConfig true
     }
```

중요한 건 아니지만 뷰바인딩과 같은 폼을 지키면 어떨까용
```suggestion
    buildFeatures {
        viewBinding = true

        //사용할 객체 가져오기 (base_url 감추는 작업)
        buildConfig = true
    }
```
요렇게 ?

---

엇! 수정했슴니당!!